pom.xml

```diff
@@ -42,13 +43,39 @@
 			<artifactId>spring-boot-starter-data-jpa</artifactId>
 		</dependency>
 
+
+		<!-- https://mvnrepository.com/artifact/org.projectlombok/lombok -->
+		<dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+			<version>1.18.24</version>
+			<scope>provided</scope>
+		</dependency>
+
+		<!-- https://mvnrepository.com/artifact/org.modelmapper/modelmapper -->
+		<dependency>
+			<groupId>org.modelmapper</groupId>
+			<artifactId>modelmapper</artifactId>
+			<version>3.1.1</version>
```

Comment:
Avoid using third-party dependencies for every single use case. It may make sense in some cases, where we don't want to reinvent the wheel like mysql-connector etc. but it comes with its own drawbacks. check: https://dev.to/brpaz/factors-to-consider-when-adding-third-party-dependencies-to-a-project-46hf

Reply:
We used ModelMapper to make object mapping easy.
So should we create custom Mapper ??